src/Domain/Creators/BinaryCompactObject.cpp

```diff
@@ -210,15 +210,15 @@ BinaryCompactObject::BinaryCompactObject(
       }
     }
   };
-  add_object_region("ObjectA", "Shell");  // 6 blocks
-  add_object_region("ObjectA", "Cube");   // 6 blocks
   add_object_region("ObjectB", "Shell");  // 6 blocks
   add_object_region("ObjectB", "Cube");   // 6 blocks
+  add_object_region("ObjectA", "Shell");  // 6 blocks
+  add_object_region("ObjectA", "Cube");   // 6 blocks
```

Comment:
can we keep this unchanged (more below)? the right-object blocks will come before left-object blocks, but I think that's good.

Reply:
Thanks @gsb76 for help with this!